CrowdStrike/pyproject.toml

```diff
@@ -8,7 +8,7 @@ authors = []
 python = ">=3.10,<3.12"
 orjson = "^3.7.7"
 python-dateutil = "^2.8.2"
-sekoia-automation-sdk = { version = "^1.8.1", extras = ["all"] }
+sekoia-automation-sdk = { version = "^1.12.2", extras = ["all"] }
```

Comment:
is ` extras = ["all"]` necessary ? for the other modules, you removed it

Reply:
It's related to whether a script is used or not. I've harmonized it.